### PR TITLE
Sanitized login page

### PR DESCRIPTION
**Pull request by JayachandraGunturu**

### CODE REVIEW

**Review by Satya-Irrinki — APPROVED** [review]

File: workspaces/client/src/LoginComponent/LoginComponent.tsx [hunk 1]
```diff
@@ -1,92 +1,79 @@
...
+import { useState } from "react";
+import { Copyright } from "../CopyrightComponent/CopyrightComponent";
+import { validateEmail } from "../Utils/validators";
+import ErrorMessages from "../Utils/ErrorMessages";
+import { LoginData } from "../Types/LoginComponentTypes";
 
-
-function mockLogin(email:any,password:any){
-  if(email === "test@gmail.com" && password === "Helloworld@123"){
+function mockLogin(email: any, password: any) {
```
Comment (venkata-sai-samanth-kommi) [inline workspaces/client/src/LoginComponent/LoginComponent.tsx:19]: remove unused mock

File: workspaces/client/src/LoginComponent/LoginComponent.tsx [hunk 2]
```diff
@@ -1,92 +1,79 @@
...
-      helper: {
-           color: 'red',
-           fontSize: '.8em',
-      }
+  const validate = (loginData: LoginData) => {
+    let validationErrors: loginErrors = loginErrors;
+    if (!validateEmail(loginData.mailid)) {
+      validationErrors.emailFormatError = ErrorMessages.emailError;
+      validationErrors.isEmailError = true;
+      setErrors({...validationErrors});
```
Comment (venkata-sai-samanth-kommi) [inline workspaces/client/src/LoginComponent/LoginComponent.tsx:56]: duplicate

**Review by venkata-sai-samanth-kommi — APPROVED** [review]

**Review by venkata-sai-samanth-kommi — APPROVED** [review]

File: workspaces/client/src/LoginComponent/LoginComponent.tsx (hunk 2 above)
Comment (JayachandraGunturu) [reply workspaces/client/src/LoginComponent/LoginComponent.tsx]: Removed duplicate code

File: workspaces/client/src/LoginComponent/LoginComponent.tsx (hunk 1 above)
Comment (JayachandraGunturu) [reply workspaces/client/src/LoginComponent/LoginComponent.tsx]: Removed mock

**Pull request merged**